lib/write-translations.js

```diff
@@ -35,6 +38,14 @@ function writeFileAndCreateFolder(file, content) {
   fs.writeFileSync(file, content);
 }
 
+function prepareTranslationText(text) {
+  if (typeof text !== 'string') {
```

Comment:
I think this `typeof` check is redundant. If `text` wasn't a string, `.trim()` would throw an error. We can do without this check 😄